api/v1alpha1/fence_module.proto

```diff
@@ -0,0 +1,20 @@
+/*
+* @Author: wangchenyu
+* @Date: 2022/2/23
+*/
+
+syntax = "proto3";
+package slime.microservice.lazyload.v1alpha1;
+
+option go_package = "slime.io/slime/modules/lazyload/api/v1alpha1";
+
+message Fence {
+  // service ports enable lazyload
+  repeated string wormholePort = 1;
+  // switch on auto generating ServiceFence for services in the namespace with label "slime.io/serviceFenced: true"
+  // default is false
+  bool disableNsReconcile = 2;
```

Comment:
I don't think users are quite sure exactly what these options do, or even where they relate to them.
We should let users configure which features to use, rather than involve them specifically in setting the detailed parameters.

Reply:
got it. I will use `disableAutoFence` to replace them